docs/usage/cli/curate/titlecase.rst

```diff
@@ -0,0 +1,9 @@
+=================
+normalize-strings
```

Comment:
Title is copy/paste typo.

Reply:
fixed in https://github.com/nextstrain/augur/commit/c00156ba7d31026aa89fc5ed68ae5215c09b99bd